python/requirements/ml/requirements_rllib.txt

```diff
@@ -32,7 +32,7 @@ higher==0.2.1
 pyglet==1.5.15
 imageio-ffmpeg==0.4.5
 # For JSON reader/writer.
-smart_open==5.0.0
+smart_open~=5.0
```

Comment:
ref https://buildkite.com/ray-project/ray-builders-pr/builds/27625#453fdb75-5f48-4f3e-a3a9-37d94a715868/6-14369
```
Cannot install smart_open==5.0.0 and smart_open==5.1.0 because these package versions have conflicting dependencies.
```

Reply:
Good catch! I would say let's remove it from tune and rllib requirements and only have it in `requirements.txt`!

Reply:
```suggestion
```